cmd/node/flags.go

```diff
@@ -511,7 +511,6 @@ func processConfigImportDBMode(log logger.Logger, configs *config.Configs) error
 		generalConfigs.GeneralSettings.StartInEpochEnabled = false
 	}
 
-	generalConfigs.StoragePruning.NumActivePersisters = generalConfigs.StoragePruning.NumEpochsToKeep
```

Comment:
we need this in a form or another:
```GO
// we need to increment this in order to make the storage resolvers work since they open 2 epochs in advance
generalConfigs.StoragePruning.NumActivePersisters++
```